src/apb/cli.py

```diff
@@ -42,14 +42,20 @@ def subcmd_init_parser(parser, subcmd):
     )
 
     subcmd.add_argument(
-        '--bindable', action='store_true', dest='bindable',
-        help=u'Make application bindable', default=False
+        '--not-bindable', action='store_false', dest='bindable',
+        help=u'Make application not bindable on the spec.', default=True
```

Comment:
@jmrodri we wanted to make the application bindable by default, with a flag to disable binding. --bindable is no longer valid but the 'bindable' variable is defaulted to True.